app.js

```diff
@@ -26,14 +34,15 @@ let run = true;
 const streamers = [];
 
 // ========================================== CONFIG SECTION =================================================================
-const CONFIG_PATH = "config.json";
+const CONFIG_PATH = "./config.json";
+let CONFIG_FILE = fs.existsSync(configPath) ? JSON.parse(fs.readFileSync(configPath, 'utf8')) : null;
 const SCREENSHOT_FOLDER = "screenshots";
 const BASE_URL = "https://www.twitch.tv";
 
 const USER_AGENT =
   "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36";
 
-const STREAMERS_URL = `${BASE_URL}/directory/game/VALORANT`;
+let STREAMERS_URL = `${BASE_URL}/directory/game/`;
```

Comment:
Why'd you make this a `let`?

Reply:
because I am adding the provided game name in main from the `CONFIG_FILE`.